test/hotspot/jtreg/runtime/ErrorHandling/HsErrFileUtils.java

```diff
@@ -131,8 +130,8 @@ public static void checkHsErrFileContent(File f, Pattern[] positivePatterns, Pat
                 lastLine = line;
                 lineNo++;
             }
-            // Found all positive pattern?
-            if (!positivePatternStack.isEmpty()) {
+            // check if current pattern is NULL then pattern matches else throw RuntimeException
```

Comment:
Sorry this doesn't read very clearly to me. Suggestion:
```
// If the current pattern is not null then it didn't match
```
Thanks

Reply:
Thank you @dholmes-ora for the suggestion. I have updated the comment. Please review the change.